usaspending_api/accounts/serializers.py

```diff
@@ -115,3 +115,10 @@ class Meta:
                 "kwargs": {"read_only": True}
             }
         }
+
+
+class ObjectClassFinancialSpendingSerializer(serializers.Serializer):
+
+    major_object_class_code = serializers.CharField()
+    major_object_class_name = serializers.CharField()
+    obligated_amount = serializers.DecimalField(21, 2)
```

Comment:
@russellbodine I think we decided in the chat to change this to `DecimalField(None, 2)` to get rid of unneeded specificity.